site/content/en/docs/concepts/cohort.md

```diff
@@ -28,7 +28,14 @@ spec:
 
 ## Configuring Quotas
 
-Similarly to [ClusterQueues](/docs/concepts/cluster_queue/#flavors-and-resources) Resource quotas may be defined at the Cohort level, and consumed by ClusterQueues within the Cohort.
+Resource quotas may be defined at the Cohort level (similarly to how they are
+defined for (ClusterQueues)[/docs/concepts/cluster_queue/#flavors-and-resources]),
+and consumed by ClusterQueues within the Cohort.  Please note that
+`nominalQuota` defined at the Cohort level represents **additional resources**
+on top of those defined by ClusterQueues within the Cohort. The Cohort's
+`nominalQuota` may be thought of as a shared pool for the ClusterQueues within
+it. Additionally, this quota may also be lent out to parent Cohort(s), subject
+to LendingLimit.
```

Comment:
I think that this is importance insight for users.
So, could we add the similar / same notes to Cohort API comment?

https://github.com/kubernetes-sigs/kueue/blob/4b7bbc9cb9afcd3b4bf5cb24b44c38fd93c2df82/apis/kueue/v1alpha1/cohort_types.go#L58